osmdroid-android/src/main/java/org/osmdroid/util/RectL.java

```diff
@@ -316,4 +316,40 @@ public static long getRotatedY(final long pX, final long pY,
                                    final double pCos, final double pSin) {
         return pCenterY + Math.round((pX - pCenterX) * pSin + (pY - pCenterY) * pCos);
     }
+
+    /**
+     * @since 6.0.2
+     */
+    public void offset(final long pDx, final long pDy) {
+        left += pDx;
+        top += pDy;
+        right += pDx;
+        bottom += pDy;
+    }
+
+    /**
+     * @since 6.0.2
+     */
+    public void union(final long pLeft, final long pTop, final long pRight, final long pBottom) {
```

Comment:
You could also write a unit test for this method.

Reply:
I assume Android's code for `Rect` is tested ;)

Reply:
:)